### PR TITLE
OC-11 Add Division Function

### PR DESCRIPTION
**Pull request by WookiefromNZ**
Added Division function, not sure how to handle divide by 0, so will throw exception 'Cannot divide be zero'. Works for Int and float 0 and 0.0 respectively.

Tidied comments.

Not sure what i was doing with my previous change (still figuring this github stuff out) so closed it and put this through again with the missing semi-colon and spaces between line 4 'a + b' fixed

### CODE REVIEW

File: OurCalculator.pde [hunk 1]
```diff
@@ -1,23 +1,21 @@
 float answer; // Will hold previously computed answer.
 
-void setup(){}
-
-void draw(){}
```
Comment (joseph-allen) [inline OurCalculator.pde]: don't delete these as I am sure we will need them shortly, although I understand why you did!

**Review by joseph-allen — CHANGES_REQUESTED** [review]

**Comment by joseph-allen** [comment]
@jorn600 you should be able to use the built in review process if you want to, it blocks my merging it in if you request changes

**Comment by WookiefromNZ** [comment]
Ah damn I don't no what i'm doing here. did it actually change? Somehow i have my own repo of the own code now and something about conflicts?

**Comment by joseph-allen** [comment]
@WookiefromNZ it's okay don't worry.

Closing the pull request does nothing :) We do however have conflicts with the main branch now you may want to look up how to resolve that. I can do it here in the GitHub GUI but it is a good thing to try to learn if you want to have a look.

**Pull request merged**